geospaas/vocabularies/managers.py

```diff
@@ -57,10 +69,12 @@ class PlatformManager(models.Manager):
     def get_by_natural_key(self, short_name):
         return self.get(short_name=short_name)
 
-    def create_from_vocabularies(self):
+    def create_from_vocabularies(self, **kwargs):
```

Comment:
The function `create_from_vocabularies` looks exactly the same in all models. What if use OOP wisely to improve that. First we crate a base class that defines the skeleton of the function:
```
class VocabularyManager(models.Manager):
    def create_from_vocabularies(self, **kwargs):
        ''' Create database instances from the pythesint list.
        '''
        num = 0
        if force==True or len(self.get_list())==0:
                self.update()
        for var in self.get_list():
            pp, created = self.get_or_create(var)
            if created: num+=1
        print("Successfully added %d new parameters" %num)
```
Then we subclass this base class for each manager and assign self.get_list and self.update the specific values:
```
class ParameterManager(VocabularyManager):
    self.get_list = pti.get_wkv_variable_list
    self.update = pti.update_wkv_variable

Then of course follow definition of other functions in the ParameterManager
```